### ISSUE TITLE
A logo for Strophe project?

### ISSUE THREAD

**Issue by Neustradamus** [strophe/libstrophe#156]
It will be nice to have a logo for Strophe project.

**Comment by pasis** [comment]
Agree, logo is a nice thing. I wonder what license should be for such logo/picture in order to let people display it in their clients? @Neustradamus do you have any idea?

**Comment by Chirunos** [comment]
> Agree, logo is a nice thing. I wonder what license should be for such logo/picture in order to let people display it in their clients? @Neustradamus do you have any idea?

I think the [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/deed.en) would be just fine.
